tests/Sniffs/WhiteSpace/DocBlock/DocBlockSniffTest.php

```diff
@@ -20,4 +20,22 @@ public function testDetection()
 		$this->assertSame(0, $codeSnifferRunner->getErrorCountInFile(__DIR__ . '/correct.php'));
 	}
 
+
+	public function testIdentationInside()
+	{
+		$codeSnifferRunner = new CodeSnifferRunner('ZenifyCodingStandard.WhiteSpace.DocBlock');
+
+		$this->assertSame(2, $codeSnifferRunner->getErrorCountInFile(__DIR__ . '/wrong-inside.php'));
+		$this->assertSame(0, $codeSnifferRunner->getErrorCountInFile(__DIR__ . '/correct-inside.php'));
+	}
+
+
+	public function testIdentationInside2()
+	{
+		$codeSnifferRunner = new CodeSnifferRunner('ZenifyCodingStandard.WhiteSpace.DocBlock');
+
+		$this->assertSame(1, $codeSnifferRunner->getErrorCountInFile(__DIR__ . '/wrong-inside2.php'));
```

Comment:
These can be added to previous method. As it's standard, that tests are per sniff.